app/views/outreach_events/show.html.erb

```diff
@@ -39,5 +44,16 @@
   <%= @outreach_event.outreach_event_types.map(&:name).join(", ") %>
 </p>
 
-<%= link_to 'Edit', edit_outreach_event_path(@outreach_event) %>
-<%= link_to 'Back', outreach_events_path %>
\ No newline at end of file
+<% if current_user.admin? || @outreach_event.user_id == current_user.id %>
+  <%= link_to 'Edit', edit_outreach_event_path(@outreach_event) %>
+  <% if @outreach_event.user_id == current_user.id %>
+    <%= link_to 'Delete', outreach_event_path(@outreach_event), method: :delete, data: { confirm: 'Are you sure?' } %>
+  <% end %>
+<% end %>
+<% if current_user.admin? %>
+  <%= link_to 'Back', admin_outreach_events_path %>
+<% elsif current_user.facilitator? %>
```

Comment:
If facilitator routing is set to be the same as student, we can remove this elsif condition